### ISSUE TITLE
No way to differentiate between IPV4/IPV6 Port Mapptings

### ISSUE THREAD

**Issue by vincentjames501** [aws/amazon-ecs-agent#3096]
### Summary

Right now with the Task metadata endpoint version 4 API, there is no way to differentiate which `Ports` are for ipv4/ipv6.

References:
https://github.com/aws/amazon-ecs-agent/issues/2870
https://github.com/aws/amazon-ecs-agent/pull/3025

### Description

We are now getting ipv4 & ipv6 ports when we were previously just getting ipv4.

We are now getting the following when querying the ECS task metadata endpoint:

```bash
$ curl http://169.254.170.2/v4/c8739a8c-7d64-44e0-8600-427ca273f031/task | jq
```
```json
{
...
      "Ports": [
        {
          "ContainerPort": 8888,
          "Protocol": "tcp",
          "HostPort": 49832
        },
        {
          "ContainerPort": 8888,
          "Protocol": "tcp",
          "HostPort": 49825
        }
      ],
...
}
```

Note docker inspect will show the HostIp:

```
$ sudo docker inspect 8507b80d2320
```
```json
{...
            "Ports": {
                "10000/tcp": null,
                "8888/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "49832"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "49825"
                    }
                ]
            },
...
}
```

### Expected Behavior

Ideally I'd love for there to be a way to differentiate these. Can we just expose HostIp such that this:

```json
{
          "ContainerPort": 8888,
          "Protocol": "tcp",
          "HostPort": 49832
},
```

Would become:

```json
{
          "ContainerPort": 8888,
          "Protocol": "tcp",
          "HostPort": 49832,
          "HostIp": "0.0.0.0"
},
```

This would give API clients the ability to differentiate easily between them.

### Observed Behavior

There is no way to distinguish between the two via the task metadata endpoint v4.

#### ECS

```
"Version":"Amazon ECS Agent - v1.56.0 (174ace6d)"
```

#### Docker

```
Client:
 Version:           20.10.7
```

**Comment by mythri-garaga** [comment]
Hi @vincentjames501,

Thank you for reporting this, we will take a look at this.

**Comment by mythri-garaga** [comment]
Closing this issue as it is addressed in #3112.
